vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -423,14 +436,21 @@ func (d *VolumeDriver) Remove(r volume.Request) volume.Response {
 	log.WithFields(log.Fields{"name": r.Name}).Info("Removing volume ")
 
 	// Docker is supposed to block 'remove' command if the volume is used. Verify.
-	if d.getRefCount(r.Name) != 0 {
+	refcnt, err := d.getRefCount(r.Name)
+
+	if err != nil {
+		log.Error(err.Error())
+		return volume.Response{Err: err.Error()}
+	}
+
+	if refcnt != 0 {
 		msg := fmt.Sprintf("Remove failure - volume is still mounted. "+
-			" volume=%s, refcount=%d", r.Name, d.getRefCount(r.Name))
+			" volume=%s, refcount=%d", r.Name, refcnt)
```

Comment:
Here you can check GetInitSuccess and return correct error by keeping the existing code as-is.